.github/workflows/releaser.yml

```diff
@@ -0,0 +1,68 @@
+name: Releaser
+on:
+  push:
+    branches-ignore:
+      - '**'
+    tags:
+      - 'v*.*.*'
```

Comment:
idk if this is a big deal or not but I switched to just `*` we might want to standardize. The reason was that this globe doens't support a full semvar that's like `v1.2.3-deadbeef+alpha`

Reply:
sg